src/sentry_tracing.c

```diff
@@ -1,4 +1,5 @@
-#include "sentry_sync.h"
+#ifdef SENTRY_PERFORMANCE_MONITORING
```

Comment:
the whole file should ideally only be compiled if the flag is set, otherwise the file is still part of the build, just completely empty.